src/main/java/net/minestom/server/entity/metadata/other/AreaEffectCloudMeta.java

```diff
@@ -37,12 +38,12 @@ public void setSinglePoint(boolean value) {
         super.metadata.setIndex(OFFSET + 2, Metadata.Boolean(value));
     }
 
-//    public ParticleWrapper getParticle() {
-//        return super.metadata.getIndex((byte) 10, new ParticleWrapper(Particle.EFFECT, null));
-//    }
-//
-//    public void setParticle(ParticleWrapper value) {
-//        super.metadata.setIndex((byte) 11, Metadata.Particle(value));
-//    }
+    public @NotNull Particle getParticle() {
```

Comment:
should just be nullable, no?

Reply:
we're passing default dust, in `getIndex` if the entry is null it will return the default.